Automattic/wp-calypso
# Theme Showcase: Add support for My Themes for Simple Sites

Related to https://github.com/Automattic/dotcom-forge/issues/4844

## Proposed Changes

* Display the `My Themes` section for Simple sites
* Populate the section with themes that were previously installed on the site

## Testing Instructions



* Apply D135351-code on your sandbox
* Use the live link and go to Theme showcase
* Notice `My themes` section is diplayed
* Notice that it's populated with themes that were previously installed on that site

## Pre-merge Checklist
